WalletWasabi.Gui/Controls/WalletExplorer/WalletExplorerViewModel.cs

```diff
@@ -186,7 +186,7 @@ internal WalletViewModelBase OpenWallet(Wallet wallet)
 
 			InsertWallet(walletViewModel);
 
-			if (!WalletManager.AnyWallet(x => x.State >= WalletState.Started && x != walletViewModel.Wallet))
+			if (!WalletManager.AnyWallet(x => x.State >= WalletState.Started && x != wallet) && wallet.State < WalletState.Stopping)
```

Comment:
```suggestion
			if (!WalletManager.AnyWallet(x => x.State == WalletState.Started && x != wallet))
```

Reply:
`wallet.State < WalletState.Stopping` is error-prone and is is not possible to understand what it means, I prefer to be explicit.

Reply:
@molnard that's not good. Notice `&& wallet.State < WalletState.Stopping` is outside `AnyWallet`.